Chummer/Classes/clsOptions.cs

```diff
@@ -399,7 +408,9 @@ static GlobalOptions()
             LoadBoolFromRegistry(ref _blnUseLogging, "uselogging");
 
             //Should the App "Phone home"
-            LoadBoolFromRegistry(ref _blnUseLoggingApplicationInsights, "useloggingApplicationInsights");
+            string myuseAI = "notset";
```

Comment:
_Consider_ (no more, this is a matter of debate) enclosing those 3 lines in a block `{}`. This will prevent `myuseAI` poluting the outer scope, at the cost of some extra indentation. 

Also, _my_useAI, I don't think the my really adds any information to the variable name, but I'm nitpicking here

Reply:
Well, the _actually_ correct solution would be `LoadEnumFromRegistry<T>`, but...